ando/engine.py

```diff
@@ -171,18 +170,21 @@ def is_AnDO_R(subpath, level, validate):
             validate.append(is_session(subpath[level]))
             is_AnDO_R(subpath, level + 1, validate)
         if level == 3:
-            validate.append(is_source(subpath[level]))
-            is_AnDO_R(subpath, level + 1, validate)
+            if(subpath[level] == "rawdata"):
```

Comment:
Hi!
This is a general comment on the code of this function, not a particular comment on this precise line... I have to admit I don't really understand your algorithm to check things level by level... It seems you do the same check at level 3 and 4 (rawdata and metadata), and a different one at level 5 (derivatives), which does not make sense to me because these 3 directories (rawdata, metadata and derivatives) are at the same level of the hierarchy... Maybe it's the function that generates the subpath that you're testing here that does bizarre things??? We should talk about it ;)